src/theme/mod.rs

```diff
@@ -69,7 +72,18 @@ impl Theme {
             ];
 
             for (filename, dest) in files {
-                load_file_contents(filename, dest);
+                if let Err(e) = load_file_contents(&filename, dest) {
+                    warn!("Couldn't load custom file, {}: {}", filename.display(), e);
+                }
+            }
+        }
+
+        // jquery is special because it can often have multiple forms
+        // ("jquery-2.1.4.min.js", "jquery.js", "jquery.min.js", etc), so we
+        // need to manually check the theme_dir for a "valid" filename.
+        if let Some(jquery) = search_dir_for_js(&theme_dir, "jquery") {
```

Comment:
I have a problem with all this jquerry related machinery. I would just remove all of it (sadly large part of this commit) and rename `jquery-2.1.4.min.js` to simple `jquery.js` we don't keep the original version numbers in the filenames for other js anyway. Also tracking the version via filename is quite brittle (most likely we could use one of the js dependency managers like npm - we use it for stylus anyway).